tests/FSharp.Test.Utilities/XunitSetup.fs

```diff
@@ -2,6 +2,10 @@ namespace FSharp.Test
 
 open Xunit
 
+/// Exclude from parallelization. Execute test cases in sequence and do not run any other collections at the same time.
```

Comment:
Yeah this is a much more descriptive name :)
Feel free to reference the xunit comment, it adds some useful context.

Reply:
Yes, the diff won't get much smaller in terms of number of files but it's manageable now, I think. There might be some last minute instability in CI because of https://github.com/dotnet/fsharp/pull/17872/commits/f838e7ee3033b34f4e097e95ad8949a274d738df, but overall this should be good now.